sdk/storage/azure-storage-blobs/test/page_blob_client_test.cpp

```diff
@@ -280,7 +280,11 @@ namespace Azure { namespace Storage { namespace Test {
     Blobs::UploadPageBlobPagesOptions options;
     ContentHash hash;
     hash.Algorithm = HashAlgorithm::Crc64;
-    hash.Value = Crc64::Hash(blobContent.data(), blobContent.size());
+
+    {
+      Crc64Hash instance;
+      hash.Value = instance.Final(blobContent.data(), blobContent.size());
```

Comment:
Nit: can we add a static method `Hash`?

Reply:
We can add it back later if we get customer feedback, and @JeffreyRichter recommended I remove it for now (same with `Md5Hash`).

How commonly do you think customers would call the `single-shot` hashing API, rather than the streaming one?